### DISCUSSION TITLE
글또 7기를 마무리하면서 | 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#29] [Announcements]
# 글또 7기를 마무리하면서 | 정현수 기술 블로그

또라이였나?

https://junghyeonsu-dev.vercel.app/posts/Geultto-7th-retrospect

**Comment by deli-ght** [comment]
티스토리로 돌아갔다면,, 아마 오늘 이 회고글은 못남기지 않았을까 생각합니다,,

**Reply by junghyeonsu** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ그렇겠네요...

**Comment by yeeeeeeeni** [comment]
👍👍👍👍
